src/Tools/ExternalAccess/FSharp/Diagnostics/DiagnosticCustomTags.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Diagnostics
+{
+    public static class DiagnosticCustomTags
+    {
+        /// <summary>
+        /// it is string[] because DiagnosticDescriptor expects string[]. 
+        /// </summary>
+        private static readonly string[] s_microsoftCustomTags = new string[] { WellKnownDiagnosticTags.Telemetry };
+        private static readonly string[] s_editAndContinueCustomTags = new string[] { WellKnownDiagnosticTags.EditAndContinue, WellKnownDiagnosticTags.Telemetry, WellKnownDiagnosticTags.NotConfigurable };
+        private static readonly string[] s_unnecessaryCustomTags = new string[] { WellKnownDiagnosticTags.Unnecessary, WellKnownDiagnosticTags.Telemetry };
+
+#pragma warning disable CA1819 // Properties should not return arrays
+        public static string[] Microsoft
```

Comment:
Can we not just reference the existing ones?

```suggestion
        public static string[] Microsoft => Microsoft.CodeAnalysis.Diagnostics.DiagnosticCustomTags.Microsoft;
```

Reply:
I can yes, I just copied and pasted it.